mobile/lib/repositories/integration.repository.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:http/http.dart' as http;
+import 'package:triggo/api/call.dart';
+import 'package:triggo/api/response.dart';
+import 'package:triggo/repositories/credentials.repository.dart';
+import 'package:triggo/repositories/dtos/integration.dtos.dart';
+
+class IntegrationRepository {
+  final http.Client? client;
+  final CredentialsRepository? credentialsRepository;
+
+  IntegrationRepository({this.client, required this.credentialsRepository});
+
+  Future<Response<OutGetIntegrationDTO>> getIntegrations() async {
+    final response = await call(
+      method: 'GET',
+      endpoint: '/user/integrations',
+      headers: {
+        'Authorization':
+            'Bearer ${await credentialsRepository!.getAccessToken()}'
+      },
+      client: client,
+    );
+
+    return Response<OutGetIntegrationDTO>(
+      statusCode: response.statusCode,
+      message: response.message,
+      headers: response.headers,
+      data: response.data != null
+          ? OutGetIntegrationDTO.fromJson(response.data!)
+          : null,
+      errors: response.errors,
+    );
+  }
+
+  Future<Response<OutGetIntegrationDTO>> getIntegrationByPage(int page) async {
+    final response = await call(
+      method: 'GET',
+      endpoint: '/user/integrations/?page=$page',
+      headers: {
+        'Authorization':
+            'Bearer ${await credentialsRepository!.getAccessToken()}'
+      },
+      client: client,
+    );
```

Comment:
I don't refuse this PR for that but it's just my opinion
```suggestion
    final accessToken = await credentialsRepository.getAccessToken();
    final response = await call(
      method: 'GET',
      endpoint: '/user/integrations/?page=$page',
      headers: {
        'Authorization':
            'Bearer ${accessToken}'
      },
      client: client,
    );
```